### PR TITLE
[rosdep] python3-numpy-stl

### PR DESCRIPTION
**Pull request by reinzor**
Debian: https://packages.debian.org/bullseye/numpy-stl
Ubuntu: https://packages.ubuntu.com/bionic/python3-numpy-stl
Fedora: https://fedora.pkgs.org/33/fedora-x86_64/python3-numpy-stl-2.11.2-2.fc33.x86_64.rpm.html

### CODE REVIEW

**Review by ivanpauno — APPROVED** [review]

File: rosdep/python.yaml [hunk 1]
```diff
@@ -6173,6 +6173,10 @@ python3-numpy:
   openembedded: [python3-numpy@openembedded-core]
   rhel: ['python%{python3_pkgversion}-numpy']
   ubuntu: [python3-numpy]
+python3-numpy-stl:
+  debian: [numpy-stl]
+  fedora: [python3-numpy-stl]
+  ubuntu: [python3-numpy-stl]
```
Comment (nuclearsandwich) [inline rosdep/python.yaml:6179]: This package does not appear to be available on Ubuntu Xenial which is still a few months away from end-of-support. I could bend on this if another rosdep maintainer decides that It's Fine:tm:.
```suggestion
  ubuntu: 
    xenial: [null]
    '*': [python3-numpy-stl]
```

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
Two questions for @reinzor 

1. Both Debian and Ubuntu appear to provide python3-numpy-stl virtual packages as well as numpy-stl packages. I think it makes sense to use the same definition on both platforms. Based on the key name I'd lean toward making the debian definition `python3-numpy-stl` as well.

2. Ubuntu Xenial does not have this package so we can't use the elided definition form. I've made a suggestion which I think is accurate and compact but I've got a weird hangup with rosdep rules and am occasionally wrong.

Once those questions are addressed this looks good to me.

**Comment by reinzor** [comment]
@nuclearsandwich thanks for the review. Didn't see the `python3-numpy-stl` package on debian before. Just checked it and updated the PR. Thanks for the Ubuntu suggestion for Xenial, accepted the change :).

File: rosdep/python.yaml [hunk 2]
```diff
@@ -6173,6 +6173,12 @@ python3-numpy:
   openembedded: [python3-numpy@openembedded-core]
   rhel: ['python%{python3_pkgversion}-numpy']
   ubuntu: [python3-numpy]
+python3-numpy-stl:
+  debian: [python3-numpy-stl]
+  fedora: [python3-numpy-stl]
+  ubuntu: 
+    '*': [python3-numpy-stl]
+    xenial: [null]
```
Comment (ivanpauno) [inline rosdep/python.yaml:6181]: should be

```suggestion
    xenial: null
```

?

Comment (ivanpauno) [reply rosdep/python.yaml:6181]: I think checkers are failing because of that

Comment (reinzor) [reply rosdep/python.yaml]: @ivanpauno thanks for that!

**Review by mabelzhang — COMMENTED** [review]
Could you add this to Debian,
```
stretch: null
```
since the package is only in Buster and Bullseye.

**Review by mabelzhang — APPROVED** [review]
Never mind! We don't need stretch anymore.

**Pull request merged**